guides/14_Content_Security_Policy.md

```diff
@@ -0,0 +1,38 @@
+# Content Security Policy for applications using the Amazon Chime SDK
+
+Modern web applications use [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to protect users from certain classes of attacks. You can add a security policy by configuring your web server to return the Content-Security-Policy HTTP header or using <meta> element to configure a policy. A guide to use a Lambda@Edge function to inject security headers through CloudFront, see [Adding HTTP Security Headers Using Lambda@Edge and Amazon CloudFront](https://aws.amazon.com/blogs/networking-and-content-delivery/adding-http-security-headers-using-lambdaedge-and-amazon-cloudfront/).
+
+## Meetings connectivity
+
+All applications that use voice and video connectivity — _e.g._, `AudioVideoFacade` classes — will need:
```

Comment:
Our README and many guides do not mention `AudioVideoFacade` because builders can access it via the meeting session object. How about the following?

> If you use a [meeting session](https://github.com/aws/amazon-chime-sdk-js#meeting-session) to enable audio, video, and screen sharing in your application, ensure that your policy includes the following:
>
> `connect-src` `*.chime.aws`